types/aws-lambda/index.d.ts

```diff
@@ -581,6 +606,7 @@ export interface CloudFrontRequest {
     uri: string;
     querystring: string;
     headers: CloudFrontHeaders;
+    origin?: CloudFrontOrigin;
```

Comment:
The docs were a little confusing here, so just to be clear, it's possible that this is not provided by cloudfront, e.g. that your handler should test if this is set before reading it?

Reply:
I'm also not 100% sure but I will check for some more API specifications and comment back.